### ISSUE TITLE
Add a Decorations quest tab

### ISSUE THREAD

**Issue by mcilhargey** [NillerMedDild/Enigmatica6#3344]
### Describe your suggestion

This tab will demonstate and show off the different decoration mods, maybe offering some interesting material stacks as a result of making some of the decoration stations. Chipped, chisel, mccaws, framed blocks, etc. 

Other quest rewards could be decoration tools, like scaffolding and such, and perhaps some more interesting materials that may be useful for tech/magic mods. 

### Why would you like this added/changed?

Lots of discoverability for tech and magic in the quest book, but not so much for the decoration and creative aspects of the pack

**Comment by NillerMedDild** [comment]
Good idea, I agree. I even started this chapter 5 months back and keep forgetting to finish it :P If anyone is willing to continue my work they're more than welcome to, otherwise I'll hopefully get it done at some point.

**Comment by Ridanisaurus** [comment]
I will work on this chapter this weekend
